cmd/maya-apiserver/cstor-operator/spc/handler.go

```diff
@@ -176,7 +177,7 @@ func validatePoolType(spc *apis.StoragePoolClaim) error {
 func validateDiskType(spc *apis.StoragePoolClaim) error {
 	diskType := spc.Spec.Type
 	// TODO: Update below snippet to make use of predicates
-	if !(diskType == string(apis.TypeBlockDeviceCPV)) {
+	if !blockdevice.SupportedDiskType[diskType] {
```

Comment:
having SupportedDiskType in blockdevice package doesn't make sense

Reply:
Move to `pkg/storagepoolclaim/v1alpha1`